src/Symfony/Component/Notifier/Message/PushMessage.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Message;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Notification\Notification;
+use Symfony\Component\Notifier\Recipient\PushRecipientInterface;
+
+/**
+ * @author Tomas Norkūnas <norkunas.tom@gmail.com>
+ */
+final class PushMessage implements MessageInterface
+{
+    private $transport;
+    private $recipientId;
+    private $subject;
+    private $content;
+    private $options;
+    private $notification;
+
+    public function __construct(string $recipientId, string $subject, string $content, MessageOptionsInterface $options = null)
+    {
+        if ('' === $recipientId) {
+            throw new InvalidArgumentException(sprintf('"%s" needs a recipient id, it cannot be empty.', static::class));
```

Comment:
```suggestion
            throw new InvalidArgumentException(sprintf('"%s" needs a recipient id, it cannot be empty.', __CLASS__));
```

Reply:
why? :) this would be inconsistent with EmailMessage, SmsMessage

Reply:
I think it's clearer with `__CLASS__` but if the other messages are like this no problem! :slightly_smiling_face: